src/admin/translations/views/TranslationsOverviewV2.module.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/variables';
+
+.disable-background {
+	opacity: 0.5;
+}
+
+.c-modal-header {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.c-translations-overview {
+	:global(.c-input) {
+		padding: 0;
+		background-color: transparent;
+		border: 0;
+		position: relative;
+		margin-top: 2rem;
+		margin-bottom: 2rem;
+
+		:global(.c-input__field) {
+			margin: 0;
+			border-radius: 0.3rem;
+			background-color: #fff;
+			border: 1px solid #bac7d1;
+			font-size: inherit;
+			height: 3.6rem;
+			padding: 0.8rem 0 0.8rem 3.2rem;
+			width: 100%;
+			appearance: none;
+			transition: all 0.15s ease-in-out;
+
+			&:focus {
+				outline: 0;
+				border-color: #25a4cf;
+				box-shadow: inset 0 0 0 0.1rem #25a4cf;
+			}
+		}
```

Comment:
Most of these colors are probably available in the variables.

edit: check other values for occurence in variables as well

Reply:
fixed